x-pack/filebeat/tests/system/test_http_endpoint.py

```diff
@@ -90,6 +90,45 @@ def test_http_endpoint_request(self):
         assert output[0]["input.type"] == "http_endpoint"
         assert output[0]["json.{}".format(self.prefix)] == message
 
+    def test_http_endpoint_cel_request(self):
+        """
+        Test http_endpoint input with HTTP events using CEL.
+        """
+        options = """
+  content_type: application/x-ndjson
+  program: |
+    {{
+      "testmessage": obj.testmessage+'_'+obj.testmessage,
+    }}
+"""
+        self.get_config(options)
+        filebeat = self.start_beat()
+        self.wait_until(lambda: self.log_contains(
+            "Starting HTTP server on {}:{}".format(self.host, self.port)))
+
+        N = 10
+        message = "somerandommessage_{}"
+        payload = "\n".join(
+            [json.dumps({self.prefix: message.format(i)}) for i in range(N)])
+        headers = {"Content-Type": "application/x-ndjson",
+                   "Accept": "application/json"}
+        r = requests.post(self.url, headers=headers, data=payload)
+
+        self.wait_until(lambda: self.output_count(lambda x: x == N))
+        filebeat.check_kill_and_wait()
+
+        output = self.read_output()
+
+        print("response:", r.status_code, r.text)
```

Comment:
Do we really need to print the response all the time? I believe it would be better to only print it on errors, this way we avoid polluting the test output.

Reply:
This reflects the behaviour in the other tests. I think deviating muddies things when these tests are finally cleaned away and replaced with Go tests.